admin_guide/cluster_metrics.adoc

```diff
@@ -226,9 +235,29 @@ items:
               args:
                 - "-source=kubernetes:https://kubernetes.default.svc?auth=&insecure=true&useServiceAccount=true"
```

Comment:
pretty sure `auth=&insecure=true` isn't right... @mwringe, do you have feedback for the right settings to use here?